selenium_tests/lti_emailer/page_objects/emailer_member_list_page_object.py

```diff
@@ -0,0 +1,32 @@
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.common.by import By
+from selenium_tests.lti_emailer.page_objects.emailer_base_page_object import EmailerBasePageObject
+
+class EmailerMainPageLocators(object):
+    PAGE_TITLE = (By.CSS_SELECTOR, "h1")
+
+
+class CourseEmailerListPage(EmailerBasePageObject):
+
+    def is_loaded(self):
+        """
+        Verifies that the page is loaded correctly by validating the title
+        :returns True if title matches else a RuntimeError is raised
+        """
+        # Note: this just checks that the  title is displayed;
+        # it doesn't guaranteed that everything we expect is rendered on the
+        # page, because angular fetches the data asynchronously
+
+        title = None
```

Comment:
Nitpick: don't need this line.  If there's an exception getting the title, we return immediately.  Otherwise, it's defined inside the `try`.